app/pages/projects-disciplines.cjsx

```diff
@@ -0,0 +1,49 @@
+counterpart = require 'counterpart'
```

Comment:
This file can be removed, please do so by merging https://github.com/zooniverse/Panoptes-Front-End/pull/2369